### ISSUE TITLE
Question about the paper

### ISSUE THREAD

**Issue by futakw** [val-iisc/BPFC#3]
I thought this paper is interesting and I'd like to ask a simple question.

Have you tried pairing logits between other types of images, not only bit-reduced images?

**Comment by SravantiAddepalli** [comment]
Thank you for your interest in our work. We tried using the proposed regularizer between clean images and images corrupted with uniform noise as well. We show the results of this ablation experiment (A2) in Table-1. We get 31.9% robust accuracy with this, whereas using quantized images we achieve 34.4%.

**Comment by futakw** [comment]
Thank you for your response!
